src/buildercore/command.py

```diff
@@ -1,17 +1,32 @@
-# Fabric 1.14 documentation: https://docs.fabfile.org/en/1.14/
-
+import os
 import fabric.api as fab_api
 import fabric.contrib.files as fab_files
 import fabric.exceptions as fab_exceptions
 import fabric.state
 import fabric.network
 import logging
 from io import BytesIO
-from . import utils
+from . import utils, threadbare
 
-LOG = logging.getLogger(__name__)
+THREADBARE = 'threadbare'
+FABRIC = 'fabric'
+
+DEFAULT_BACKEND = THREADBARE
```

Comment:
could merge with Fabric first, and rollout Threadbare as the second step?

Reply:
there is already a PR for that: https://github.com/elifesciences/builder/pull/593

I opened to ensure the Fabric backend continued to work while I juggled the two. We can merge that one first if you'd like, it's a branch of this branch.